<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>property_PointToPlanMeasure</class>
 <widget class="QWidget" name="property_PointToPlanMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>579</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Point to plane properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>577</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QLabel" name="PtPDist_2">
         <property name="text">
          <string>Horizontal</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleEdit" name="HorizontalInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ValuesLabel">
         <property name="text">
          <string>Values</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="PtPDist">
         <property name="text">
          <string>Point to plane distance</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="GenericPropertiesFeet" name="genericPropsFeetWidget" native="true"/>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleEdit" name="PtoPDistInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="GenericPropertiesHead" name="genericPropsHeadWidget" native="true"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="PtPDist_3">
         <property name="text">
          <string>Vertical</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QDoubleEdit" name="PointCoordXInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDoubleEdit" name="NormalToPlaneZInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleEdit" name="ProjPointXInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleEdit" name="PointCoordZInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleEdit" name="NormalToPlaneXInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleEdit" name="PointCoordYInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="XLabel">
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="ZLabel">
           <property name="text">
            <string>Z</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QDoubleEdit" name="NormalToPlaneYInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QDoubleEdit" name="ProjPointYInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="PointCoordLabel">
           <property name="text">
            <string>Point coordinates</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleEdit" name="PointOnPlaneZInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QDoubleEdit" name="ProjPointZInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="YLabel">
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="NormalToPlaneLabel">
           <property name="text">
            <string>Normal to plane</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="PointOnPlaneLabel">
           <property name="text">
            <string>Point on plane</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleEdit" name="PointOnPlaneYInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleEdit" name="PointOnPlaneXInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="ProjectedPointLabel">
           <property name="text">
            <string>Projected point</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleEdit" name="VerticalInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="PtPDist_4">
         <property name="text">
          <string>Proj. point to plane distance</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleEdit" name="ProjPtoPDistInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenericPropertiesHead</class>
   <extends>QWidget</extends>
   <header location="global">GenericPropertiesHead.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GenericPropertiesFeet</class>
   <extends>QWidget</extends>
   <header>GenericPropertiesFeet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>PtoPDistInfield</tabstop>
  <tabstop>ProjPtoPDistInfield</tabstop>
  <tabstop>HorizontalInfield</tabstop>
  <tabstop>VerticalInfield</tabstop>
  <tabstop>PointCoordXInfield</tabstop>
  <tabstop>PointCoordYInfield</tabstop>
  <tabstop>PointCoordZInfield</tabstop>
  <tabstop>PointOnPlaneXInfield</tabstop>
  <tabstop>PointOnPlaneYInfield</tabstop>
  <tabstop>PointOnPlaneZInfield</tabstop>
  <tabstop>NormalToPlaneXInfield</tabstop>
  <tabstop>NormalToPlaneYInfield</tabstop>
  <tabstop>NormalToPlaneZInfield</tabstop>
  <tabstop>ProjPointXInfield</tabstop>
  <tabstop>ProjPointYInfield</tabstop>
  <tabstop>ProjPointZInfield</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
